### DISCUSSION TITLE
File Not Found Exception

### DISCUSSION THREAD

**Discussion by heavenchrist** [konnco/filament-import#161] [Q&A]
This is the error i get although the actually exist
File [C:\Users\user\Desktop\projects\bos\storage\app\filament-import\01J6F91WWAFN7012JNHXHTYRBW.xlsx] (local) does not exist and can therefor not be imported.

**Comment by irwansaktip** [comment]
I have same issue.
I found a miss on  `/vendor/maatwebsite`  where the code trying to check file exists on temp folder.
you can find the file on folder `/vendor/maatwebsite/excel/src/Files/TemporaryFile.php` and add 1 line of code to solve the cheking file 
on line 61.

like image bellow :
![image](https://github.com/user-attachments/assets/f8317108-9fa7-470d-9067-9e35d6dd96c9)

**Comment by tiagof** [comment]
Hi. Got this bug too but it was fixed within minutes.
Do a `composer update maatwebsite/excel` and there error should be gone.
